projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/recommendations/row-types/TableRowHover.svelte

```diff
@@ -9,7 +11,14 @@
 
 	{#if edit_url}
 		<div class="button-container">
-			<Button small fill href={edit_url}>Edit Page</Button>
+			<Button
+				small
+				fill
+				on:click={() =>
+					recordBoostEventAndRedirect( edit_url, 'clicked_edit_page_on_isa_report', {} )}
```

Comment:
Same as the above suggestion for device type. I think it's a good idea to include that in here.